test.sh

```diff
@@ -7,52 +7,131 @@ GREEN="\e[32m"
 MAGENTA="\e[95m"
 RESET="\e[0m"
 
+# This function will be called when running this script
+function main(){
+    # This function will be called when running this script
+    run_tests_in_development;
+    run_tests_in_production;
+}
 
-# development or production?
-if [[ $(ls -l docker-compose.yml) == *"production.yml"* ]]; then
```

Comment:
I did understand that line, but not why exactly it is needed. It is simple to re-enable, and since it seems to ease deployment, this seems reasonable.